test/source/tests/tests/compose.ts

```diff
@@ -25,6 +25,7 @@ export const defineComposeTests = (testVariant: TestVariant, testWithNewBrowser:
       await inboxPage.waitAll('iframe');
       const composeFrame = await inboxPage.getFrame(['compose.htm']);
       const initialComposeFrameHeight = await inboxPage.getOuterHeight('iframe');
+      composeFrame.waitAll('#section_header');
```

Comment:
you have to `await` this or it will have no effect

Basically everything that has to do with the browser has to be awaited

Reply:
oops, stupid mistake :man_facepalming: 

In ESLint there's the https://eslint.org/docs/rules/require-await rule which can be helpful for such cases.

Offtopic: did you think about using ESLint for `flowcrypt-browser`. `tslint` has been deprecated in favor of ESLint: https://medium.com/palantir/tslint-in-2019-1a144c2317a9

ESLint is truly amazing tool for keeping TS/JS in good and consistent shape and preventing silly human mistakes like this one with missing `await`.